src/Core/ObjectRegistry.php

```diff
@@ -66,25 +66,27 @@
      * @param array $config Additional settings to use when loading the object.
      * @return mixed
      */
-    public function load($objectName, $config = [])
+    public function load($objectName, array $config = [])
```

Comment:
Those are not typehints, though. So this could also break existing apps. We need to stick to the non-typehint for 3.0 and we can rethink it for 3.1.